src/app/pcpolicy/pcpolicy.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from '@angular/core';
+import { DashboardService } from '../services/dashboard.service';
+
+@Component({
+  selector: 'app-pcpolicy',
+  templateUrl: './pcpolicy.component.html',
+  styleUrls: ['./pcpolicy.component.scss']
+})
+export class PcpolicyComponent implements OnInit {
+  email: String;
+  public position= 'col-md-10 col-md-offset-2 pcpolicybody';
+  constructor(private _dashboardService: DashboardService) { }
+
+  ngOnInit() {
+    this._dashboardService.getUserInfo().subscribe(response => {
+        this.email = response.user.email;
+    });
+  }
+
+  toggle() {
+    if (this.position === 'col-md-10 col-md-offset-2 pcpolicybody text-center') {
+      this.position = 'col-md-12 pcpolicybody text-center';
+    }else {
+      this.position = 'col-md-10 col-md-offset-2 pcpolicybody text-center';
+    }
+  }
+
+  mail() {
+    this._dashboardService.mailpcpolicy().subscribe(response => {
+      // initiate pop up
```

Comment:
the thing which is left to add is a notification that email was sent.  The Notification thing has to be added at various places in the whole app, once I start I will have to add one line here which initiates that notification.